exist-core/src/main/java/org/exist/backup/ZipArchiveBackupDescriptor.java

```diff
@@ -149,6 +155,30 @@ public BackupDescriptor getBackupDescriptor(String describedItem) {
         return bd;
     }
 
+    @Override
+    public List<BackupDescriptor> getChildBackupDescriptors() {
+        final Pattern ptnDescriptor = Pattern.compile("(" + base + "/[^/]+/" + ")" + BackupDescriptor.COLLECTION_DESCRIPTOR);
+        final Matcher mtcDescriptor = ptnDescriptor.matcher("");
+        try (final Stream<BackupDescriptor> entries = archive.stream()
```

Comment:
Would it make sense to add some logging for this step (assuming this could consume some time)

Reply:
I added as requested :-)